corio.py

```diff
@@ -191,6 +193,7 @@ def schedule_execution_plan(parsed_input: dict, options: munch.Munch):
                       "endpoint_url": S3_CFG.endpoint,
                       "use_ssl": S3_CFG.use_ssl,
                       "seed": options.seed}
+    os.environ["sequential_run"] = str(options.sequential_run)
```

Comment:
Why this needs to be part of environment variable ? Any specific requirement.

Reply:
We are using it in different modules, that's reason added it in environment variable.

Reply:
I don't think its a right way to do.